examples/basic-jsx-external/index.html

```diff
@@ -10,12 +10,16 @@
     <h1>Basic Example with External JSX</h1>
     <div id="container">
       <p>
-        If you can see this, React is not working right. This is probably because you&apos;re viewing
-        this on your file system instead of a web server. Try running
+        If you can see this, React is not working right. This is probably because you&apos;re viewing this on your file system instead of a web server.<br>
+        Try navigating to the React root 
+        <pre>
+          cd /path/to/react
+        </pre>
+        and running
         <pre>
           python -m SimpleHTTPServer
         </pre>
-        and going to <a href="http://localhost:8000/">http://localhost:8000/</a>.
+        and going to <a href="http://localhost:8000/examples/basic-jsx-external/">http://localhost:8000/examples/basic-jsx-external/</a>.
```

Comment:
The problem for me is that when running `python ... ` from the directory where the file is, the example doesn't work because the server doesn't find directories below the server root (i.e. the files in `../../build/` and in `../shared/css/`). When running the server from the repository root, the example does work.

This is on Mac OS 10.10.